lib/new_embed.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library newembed;
```

Comment:
Generally, I don't think we need `library` directives.

Reply:
Removed.